wp-content/themes/wprig/template-parts/content-nineteeneighty.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Template part for displaying posts
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package veryserious
+ */
+
+?>
+
+		<?php
+		if ( is_singular() ) :
+			the_title( '<h1 class="entry-title">', '</h1>' );
+		else :
+			the_title( '<h2 class="entry-title"><a href="' . esc_url( get_permalink() ) . '" rel="bookmark">', '</a></h2>' );
+		endif;
+
+		if ( 'post' === get_post_type() ) :
+			?>
+			
+			<?php
+				veryserious_posted_on();
+				veryserious_posted_by();
+				veryserious_comments_link();
+			?>
+		
+			<?php
+		endif;
+		?>
+		<?php
+
+		$htmlcontent = get_the_content(
+				sprintf(
+					wp_kses(
+						/* translators: %s: Name of current post. 
+						Only visible to screen readers */
+						__( 'Continue reading<span class="screen-reader-text"> "%s"</span>', 
+							'veryserious' ),
+						array(
+							'span' => array(
+								'class' => array(),
+							),
+						)
+					),
+					get_the_title()
+				)
+		);
+
+		// strip everything BUT the tags here
+		$htmlcontent_0 = 
+			strip_tags( 
+				$htmlcontent , 
+				'<p><br><li><ul><ol><a><h2><h3><h4><h5><h6><i><b><u><em><strong><span>' 
+			);
+
```

Comment:
Clarity this section about stripping out empty tags, especially with what the array table is doing.